must-call-checker/src/main/java/org/checkerframework/checker/mustcall/NotOwning.astub

```diff
@@ -17,13 +17,13 @@ class FileChannel {
 
 class SocketChannel extends AbstractSelectableChannel implements ByteChannel, ScatteringByteChannel, GatheringByteChannel, NetworkChannel {
     protected SocketChannel(SelectorProvider arg0);
-    static SocketChannel open() throws IOException;
-    static SocketChannel open(SocketAddress arg0) throws IOException;
+    @MustCallChoice static SocketChannel open(@MustCallChoice SocketChannel this) throws IOException;
+    @MustCallChoice static SocketChannel open(@MustCallChoice SocketChannel this, SocketAddress arg0) throws IOException;
```

Comment:
These annotations aren't correct, and produce stub file parse errors.

These static methods open new `SocketChannel`s - the former opens an `@Unconnected` socket and the latter opens a connected socket (the unconnected socket checker has a stub for them already).

Reply:
You're right, open will create a new socket from the client side.